src/plugins/git/createGraph.js

```diff
@@ -103,21 +106,21 @@ class GitGraphCreator {
     const result = new Graph().addNode(treeNode);
     Object.keys(tree.entries).forEach((name) => {
       const entry = tree.entries[name];
-      const entryNode = {
+      const entryNode: Node<TreeEntryNodePayload> = {
         address: this.makeAddress(
           TREE_ENTRY_NODE_TYPE,
           treeEntryId(tree.hash, entry.name)
         ),
-        payload: {},
+        payload: {tree: tree.hash, name},
       };
-      const entryEdge = {
+      const entryEdge: Edge<IncludesEdgePayload> = {
```

Comment:
Why is the tree hash in the entryNode payload? It should be in the treeNode payload. Then if you need the tree hash for an entry, you get it via porcelain over:
`graph.neighborhood(entryNode, {nodeType: "TREE", edgeType: "INCLUDES_EDGE", direction: "IN"})[0].hash`

Reply:
I included this mostly for consistency (putting both relevant fields in). You're right that it is easily available from the tree, and I didn't end up using it. I'd be happy to remove it if you like.

Reply:
Actually, it isn't available in the tree ([see here]). Could you remove it from the entryNodePayload and add it to the treeNodePayload?

[see here]: https://github.com/sourcecred/sourcecred/pull/184/files#diff-080791a1f5043a49563a9556411954e2L101

Reply:
It is not available in the payload, but it is available as the (literal) address ID. You raise a fair criticism: "if you're against parsing IDs in general, this shouldn't change just because the parsing function is the identity function in some cases." I agree. But, instead of putting in the work to expose everything on payloads, I think that I'd rather wait until we have some more direction on #190.